WordPress/gutenberg
# Use block label and icon for the inserter draggable chip.

## What?

Use the block title and icon for the draggable chip, to give context of the type of block that's being inserted.

Fixes https://github.com/WordPress/gutenberg/issues/51010

## Why?


The draggable chip wasn't giving the context of the block that was being inserted. Instead, it was showing "1 block" as text, without any icon, and it was having two lines when the string contained more than one word:

## Testing Instructions

1. Create a post/page.
2. Open the block inserter.
3. Try inserting a pattern, and confirm that the inserter is showing "Pattern" as before.
4. Try inserting an image or video, and see that the block icon is being shown in the draggable chip. 

## Screenshots or screencast 

**Before**

https://github.com/WordPress/gutenberg/assets/548849/2afe05db-0be1-48e7-a957-4eddfb571a3e

**After**

https://github.com/WordPress/gutenberg/assets/252415/c9be9bc0-b47d-4acc-8bdc-edfb5074c72a
